test/unit/stats.js

```diff
@@ -0,0 +1,793 @@
+// TODO Add before each that reseeds the DB
+// Set the env to development
+process.env.NODE_ENV = 'development';
+
+// Require the testing dependencies
+var chai = require('chai');
+var assert = chai.assert;
+
+// The file to be tested
+const stats = require('../../routes/stats');
+
+// Create fake stats
+var fakeStat = {
+  stat_id: 1,
+  student_id: 1,
+  event_id: 1,
+  height: 5,
+  weight: 5,
+  pacer: 5
+};
+
+var fakeStat2 = {
+  stat_id: 2,
+  student_id: 1,
+  event_id: 2,
+  height: 7,
+  weight: 7,
+  pacer: 7
+};
+
+var fakeStat3 = {
+  stat_id: 3,
+  student_id: 2,
+  event_id: 6,
+  height: 71,
+  weight: 17,
+  pacer: 57
+};
+
+var fakeStat4 = {
+  stat_id: 4,
+  student_id: 2,
+  event_id: 4,
+  height: 40,
+  weight: 12,
+  pacer: 500
+};
+
+var fakeStat5 = {
+  stat_id: 5,
+  student_id: 2,
+  event_id: 2,
+  height: 44,
+  weight: 16,
+  pacer: 500
+};
+
+var fakeStat6 = {
+  stat_id: 6,
+  student_id: 4,
+  event_id: 2,
+  height: 4,
+  weight: 12,
+  pacer: 421
+};
+
+// This stat is added later
+var fakeStat7 = {
+  stat_id: 7,
+  student_id: 2,
+  event_id: 2,
+  height: 320,
+  weight: 54,
+  pacer: 382,
+};
+
+// Update Checking
+var fakeStat8 = {
+  stat_id: 1,
+  student_id: 1,
+  event_id: 1,
+  height: 6,
+  weight: 6,
+  pacer: 6
+};
+
```

Comment:
add this here:
// Add before each to reset database between tests
beforeEach(function() {
  return seed();
});